### ISSUE TITLE
js: Uncaught SyntaxError: Unexpected string

### ISSUE THREAD

**Issue by clawoflight** [jeena/FeedTheMonkey#10]
The window opens properly, but only displays an empty layout.

This is with feedthemonkey `2.1.2-1` and QT `5.7.0-1` on Arch Linux.

**Comment by jeena** [comment]
Oh, that is weird, I had that exact behaviour with 2.1.1 and that is why I did https://github.com/jeena/FeedTheMonkey/commit/cca7f55760f59dc4ada70ac866ab129b64e649db which is now 2.1.2, are you sure you're using 2.1.2 and not 2.1.1?

Could you start it from the terminal and post the error messages here?

**Comment by clawoflight** [comment]
I am sure:

```
pacman -Qi feedthemonkey | grep Version
Version         : 2.1.2-1
```

If I compile 2.1.1, I see the error that you fixed in cca7f55. This is a different error.

This is the output:

```
0701/154449:ERROR:browser_main_loop.cc(217)] Running without the SUID sandbox! See https://chromium.googlesource.com/chromium/src/+/master/docs/linux_suid_sandbox_development.md for more information on developing with the sandbox on.
js: Uncaught SyntaxError: Unexpected string
js: Uncaught SyntaxError: Unexpected string
```

**Comment by jeena** [comment]
It doesn't even tell us the line number ^^

Hm it's difficult to debug for me because I can't see that behaviour here, I also run it on Arch and Qt 5.7, what you could do to narrow it down is to slowly comment out one of the lines which I changed in https://github.com/jeena/FeedTheMonkey/commit/cca7f55760f59dc4ada70ac866ab129b64e649db after another and check when this happens.

**Comment by clawoflight** [comment]
The errror disappears after commenting out line 61 in `Content.qml`.

**Comment by jeena** [comment]
I see, could you remove the part with the `fontLabel.font.family` and test if it then works? It will have a different font though.

**Comment by clawoflight** [comment]
Just from playing around with it, I can't find a way. Whenever I get rid of the error, text is completely invisible.

**Comment by jeena** [comment]
Could you try to download their Qt SDK from their website and compile it with that?

**Comment by jeena** [comment]
I have been able to get rid of all the JavaScript errors in https://github.com/jeena/FeedTheMonkey/commit/99308a98c9b53fe8425c4dd116dd064160a15500 could you retest if that works for you now?

**Comment by clawoflight** [comment]
So... I had the same problem with the several commits I tried since my last message. I even did a `git  bisect` once, and it didn't find a single good revision. I was too lazy to try the Qt SDK from the website.

However, when I rebased my git repo on yours today after your last comment, and it still didn't work, I was so puzzled that I tried deleting the config file. It worked afterwards!

I have no clue what could possibly have been wrong with that file since it worked before, but oh well...
I will close this issue now as I don't think that problem will be reproducible. (I'll come back if it happens again :D )

@jeena I like the dark theme, but I it currently lacks contrast in the sidebar. Selected/read/unread posts are almost impossible to differentiate :)
